fastscanner/benchmarks/benchmark_polygon_realtime.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+import logging
+import time
+import traceback
+
+from fastscanner.adapters.realtime.polygon_realtime import PolygonRealtime
+from fastscanner.adapters.realtime.redis_channel import RedisChannel
+from fastscanner.pkg import config
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class BenchmarkStats:
+    def __init__(self):
+        self.total_records = 0
+        self.total_flushes = 0
+        self.latencies = []
+
+    def record_flush(self, record_count: int, latency: float):
+        self.total_records += record_count
+        self.total_flushes += 1
+        self.latencies.append(latency)
+
+    def report(self):
+        avg_latency = sum(self.latencies) / len(self.latencies) if self.latencies else 0
+        max_latency = max(self.latencies) if self.latencies else 0
+        print("\n--- Benchmark Report ---")
+        print(f"Total Records Pushed: {self.total_records}")
+        print(f"Total Flushes: {self.total_flushes}")
+        print(f"Avg Flush Latency: {avg_latency:.6f}s")
+        print(f"Max Flush Latency: {max_latency:.6f}s")
+
+
+def wrap_flush(redis_channel: RedisChannel, stats: BenchmarkStats):
+    original_flush = redis_channel.flush
```

Comment:
We don't want to test the flush or the push performance. We want to test the whole handle_messages performance. What you need is to wrap the handle_messages method